test/JuMPExtension.jl

```diff
@@ -49,10 +49,51 @@ function MOI.delete!(m::MyModel, vref::MyVariableRef)
     delete!(m.varnames, vref.idx)
 end
 MOI.isvalid(m::MyModel, vref::MyVariableRef) = vref.idx in keys(m.variables)
+JuMP.numvar(m::MyModel) = length(m.variables)
```

Comment:
Why do we have a method called `numvar`? This needs to change to `num_variables` or something similar.